src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -147,6 +149,36 @@ private void processRequests(HttpServerRequest httpServerRequest) {
 
                 break;
 
+            //create a sink endpoint and initialize consumer
+            case CREATE:
+                SinkBridgeEndpoint<?,?> sink = new HttpSinkBridgeEndpoint<>(this.vertx, this.bridgeConfigProperties);
+                sink.closeHandler(s -> {
+                    this.endpoints.get(httpServerRequest.connection()).getSinks().remove(s);
+                });
+
+                sink.open();
+
+                //add sink to list
+                this.endpoints.get(httpServerRequest.connection()).getSinks().add(sink);
+
+                sink.createConsumer(new HttpEndpoint(httpServerRequest));
```

Comment:
instead of having this "ad hoc" call on `sink` I would always use the pattern to call `sink.handle` passing the `HttpEndpoint`. The `handle` method inside the `sink` (which is empty in your PR) is then in charge to understand (one more time) that it's `CREATE` or `SUBSCRIBE` or `POLL` (or whatever when you'll implement getting messages) and extracting parameters and then calling `initConsumer`, `subscribe` that are in the `SinkBridgeEndpoint` base class (that you should change a little bit to avoid registering the handler for getting messages because we need poll).